src/server/scripts/Commands/cs_cheat.cpp

```diff
@@ -175,6 +175,8 @@ class cheat_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_CT, player->GetCommandStatus(CHEAT_CASTTIME) ? enabled : disabled);
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_POWER, player->GetCommandStatus(CHEAT_POWER) ? enabled : disabled);
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_WW, player->GetCommandStatus(CHEAT_WATERWALK) ? enabled : disabled);
+        handler->PSendSysMessage(LANG_COMMAND_CHEAT_TAXINODES, player->isTaxiCheater() ? enabled : disabled);
+        
```

Comment:
The only way I could have changed anything is if you added me as "collaborator" in your repository, giving me write access to your files. It was meant as a friendly comment.